java/daikon/inv/unary/stringsequence/SequenceFixedLengthString.java

```diff
@@ -0,0 +1,144 @@
+package daikon.inv.unary.stringsequence;
+
+import daikon.PptSlice;
+import daikon.inv.DiscardInfo;
+import daikon.inv.Invariant;
+import daikon.inv.InvariantStatus;
+import daikon.inv.OutputFormat;
+import org.checkerframework.checker.interning.qual.Interned;
+import org.checkerframework.checker.lock.qual.GuardSatisfied;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.dataflow.qual.Pure;
+import org.checkerframework.dataflow.qual.SideEffectFree;
+import org.checkerframework.framework.qual.Unused;
+import typequals.prototype.qual.Prototype;
+
+/**
+ * Indicates that all the elements of an array of strings have a fixed length n. Prints as {@code
```

Comment:
What is the semantics of SequenceFixedLengthString (and other invariants) with respect to `null` sequence elements?  Should SequenceFixedLengthString be true of ["aaa", "bbb", null, "ccc"]?
Please document the answer and test it.
Currently, the SequenceFixedLengthString code crashes when it encounters this, and other invariants may have the same problem.